packages/ses/index.d.ts

```diff
@@ -0,0 +1,112 @@
+type safetyPreference = 'safe' | 'unsafe';
+
+type LockdownOpts = {
+  fringeSet?: {
+    add?: Function,
+    has?: Function,
```

Comment:
If `fringSet` is present, I assume it must implment `add` and `has`. Should these `?` be here?